project/Dependencies.scala

```diff
@@ -12,10 +13,10 @@ object Dependencies {
   lazy val circe = "io.circe" %% "circe-core" % circeVersion
   lazy val dynamoDB = "software.amazon.awssdk" % "dynamodb" % awsSdkVersion
   lazy val eventBridgeSdk = "software.amazon.awssdk" % "eventbridge" % awsSdkVersion
-  lazy val mockito = "org.mockito" %% "mockito-scala" % "1.17.31"
+  lazy val mockito = "org.scalatestplus" %% "mockito-5-10" % s"$scalaTestVersion.0"
```

Comment:
How come there is a `.0` at the end?

Reply:
It tracks the scalatest versions I think but it's published with this zero on the end. I'm not sure why.